lib/edge-runtime/angelaUtils.ts

```diff
@@ -138,3 +138,8 @@ export function replacePlaceholdersDuringStreaming(
   }
   return { content, placeholderBuffer };
 }
+
+export function stripExampleFunctions(rawOutput: string): string {
```

Comment:
I think a short docstring here to remind why these specific functions are being cut would be v useful